services/backend-next/README.md

```diff
@@ -0,0 +1,24 @@
+# [Backend next](https://github.com/SciCatProject/scicat-backend-next)
+
+The SciCat backend-next is a rewrite of the original backend, built on top of the NestJS framework.
+
+## Configuration options
+
+The backend-next service is mainly configured via environment variables. For an extensive list of available options see [here](https://scicatproject.github.io/documentation/Development/v4.x/backend/configuration.html).
+
+It can also serve the frontend theme and configuration by mounting the files in `/home/node/app/dist/config/frontend.theme.json` and `/home/node/app/dist/config/frontend.config.json`.
```

Comment:
I am unsure if we should say this, as this only works if the be and fe are served under the same domain (if I understood right). So this will not work here, unless we change the traefik labels